connection.js

```diff
@@ -616,16 +647,19 @@ class Connection {
                  }
                 // now just before TimecontrolString is being tested, we again make sure it has the latest value
                 universalPeriodtimeTimecontrolString = String(t.time_control);/*
-                "fischer", "byoyomi", "canadian" */
-
-                // sanity check : if not fischer, not byoyomi, not canadian
-                if ((universalPeriodtimeTimecontrolString !== "fischer") && (universalPeriodtimeTimecontrolString !== "byoyomi") && (universalPeriodtimeTimecontrolString !== "canadian")) {
-                    conn_log ("error, could not find time control in " + t.time_control);
-                    return { reject : true, msg: "error, could not find time control in " + t.timecontrol};
+                "fischer", "byoyomi", "canadian", "simple" */
+
+                // sanity check : if not fischer, not byoyomi, not canadian, not simple
+                // time control peculiarities : 
+                // - fischer : doesnt have a minperiods or maxperiods
+                // - simple : doesnt have a maintime, only periodtime
+                if ((universalPeriodtimeTimecontrolString !== "fischer") && (universalPeriodtimeTimecontrolString !== "byoyomi") && (universalPeriodtimeTimecontrolString !== "canadian") && (universalPeriodtimeTimecontrolString !== "simple")) { // simple is part of allowed timecontrols
+                    conn_log ("error, could not find allowed time control in " + t.time_control);
+                    return { reject : true, msg: "error, could not find allowed time control in " + t.timecontrol};
                 }                
 
                 // if sanity check passes :
-                if ((universalPeriodtimeTimecontrolString === "fischer") || (universalPeriodtimeTimecontrolString === "byoyomi") || (universalPeriodtimeTimecontrolString === "canadian")) {
+                if ((universalPeriodtimeTimecontrolString === "fischer") || (universalPeriodtimeTimecontrolString === "byoyomi") || (universalPeriodtimeTimecontrolString === "canadian") || (universalPeriodtimeTimecontrolString === "simple")) {
```

Comment:
Do we really need this if at all here? The code below all also checks for `universalPeriodtimeTimecontrolString === xxx` in sub ifs, so this adds little value.

Reply:
i dont remember, but we'd need to review that in UHMAEAT 2.2 at the end of #129 , same

Reply:
The whole point of small step commits is to introduce non-buggy and clean clear code among each step. When we review the next one, this change wont be visible anymore.